source/administration/console/subnet-registration.rst

```diff
@@ -0,0 +1,74 @@
+.. _minio-console-subscription:
+
+===================
+SUBNET Registration
+===================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+The MinIO Console provides a graphical user interface for several Subscription related functions.
+Use these functions to subscribe, manage the deployment's |SUBNET| license, or access Support tools for sharing with MinIO Engineering.
+
+License
+-------
+
+The :guilabel:`License` section displays information on the licensing status of the MinIO deployment.
+
+For deployments not registered via |subnet|, the Console displays a table comparison of MinIO License and Support plans:
+
+Existing customers can register the deployment with their |subnet| account by clicking :guilabel:`Register this cluster` in the top-right corner of the screen.
+
+MinIO is Open Source software under the :minio-git:`GNU AGPLv3 license <mc/blob/master/LICENSE>`.
+Applications using MinIO should follow local laws and regulations around licensing to ensure compliance with the AGPLv3 license, which may include open sourcing the application stack.
+
+Proprietary application stacks can register for either the SUBNET :guilabel:`Standard` or :guilabel:`Enterprise` License and Support plan to use MinIO under a commercial license.
+
+Support
+-------
+
+The :guilabel:`Support` section provides an interface for generating health and performance reports.
+Support functionality requires registering your deployment with |subnet|. 
+Unregistered deployments display a :guilabel:`Register Your Cluster` button to register with your |subnet| account.
+See the :guilabel:`License` section in the Console or visit the `MinIO SUBNET <https://min.io/pricing?jmp=docs>` website for more information on registration.
+
+This section contains the following subsections.
+Some subsections may not be visible if the authenticated user does not have the :ref:`required administrative permissions <minio-policy-mc-admin-actions>`.
```

Comment:
This...may not apply for support tools? I'm actually not sure what admin permissions we require here. For health three's one, but the performance testing tools might be PUT/GET/LIST.

Maybe drop this section for now and revisit if/when we can delineate exactly what permissions we require.

Reply:
removed